deltachat-repl/README.md

```diff
@@ -0,0 +1,60 @@
+# Delta Chat REPL
+
+This is a simple [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) frontend build on top of delta chat core.
+It's purpose is to help with quick testing during development, it is not meant for end users.
+
+Dependencies:
+- if you want to use `getqr` you need `qrencode` (macOS: `brew install qrencode`)
```

Comment:
In Debian the package is also `qrencode`, and i think, everywhere, so per-OS instructions may be omitted.

Reply:
```suggestion
- if you want to use `getqr` you need `qrencode` (To install use your systems package manager)
```